src/SweetAlert/functions.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+// test comment
```

Comment:
Hmm interesting. Can you do: 
```suggestion
// test comment v2.0
```